index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <link rel="stylesheet" href="style.css">
+
+    <div class="container">
```

Comment:
[ ] kindly use the `header` tag for your page header, it should group the site's title or logo (or both) and the main site navigation 🤓.[check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.